doc/nrf/releases_and_maturity/migration/migration_guide_3.2.rst

```diff
@@ -29,8 +29,33 @@ Samples and applications
 
 This section describes the changes related to samples and applications.
 
-|no_changes_yet_note|
+Matter
+------
+
+.. toggle::
+
+   * For the Matter samples and applications:
+
+      * All Matter over Wi-Fi samples and applications now store a portion of the application code related to the nRF70 Series Wi-Fi firmware in external flash memory by default.
```

Comment:
I think we need to mention directly that as a consequence performing DFU between older NCS revision and 3.2.0 will not be possible without disabling storing fw patch on ext flash and using previous partitions layout.

Reply:
Added, please take a look.